components/form/FormItem.tsx

```diff
@@ -24,22 +24,17 @@ type ChildrenType = RenderChildren | React.ReactNode;
 interface MemoInputProps {
   value: any;
   update: number;
-  children: any;
+  children: React.ReactNode;
 }
 
-const MemoInput = React.memo<MemoInputProps>(
-  ({ children }) => {
-    return children;
-  },
+const MemoInput = React.memo(
+  ({ children }: MemoInputProps) => <>{children}</>,
```

Comment:
Why create a fragment to wrap this?

Reply:
@fa93hws ping

Reply:
It can be `as JSX.Element` to pass the type check if the fragment is not preferred.